cabinets/plugins.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+import importlib.util
+import os
+import pkgutil
+
+from cabinets.logger import info
+
+
+def discover_built_in(plugin_path):
```

Comment:
Seems like there should be parity between the two `discover_` methods, either taking a single `plugin_path` or `*paths`.